backend/app/verification.py

```diff
@@ -295,17 +312,102 @@ def get_available_methods(appid: str, login=Depends(login_state)):
 
     if provider := _get_provider_username(appid):
         provider_name, username = provider
+        organization = False
+
+        if provider_name == LoginProvider.GITHUB:
+            try:
+                user = github.Github().get_user(username)
```

Comment:
Shouldn't we be able to use `get_user()` instead or are we not logged in as that user?
Or is this an unauthorized call? If yes it might make sense to authorize due to api limits

Reply:
`username` is from the app ID. We won't be logged in under that name if it belongs to an organization. We could still authorize the call with the current user's token, though.

Reply:
I changed some of this code around when I implemented the frontend.